core/src/main/java/apoc/ApocSettings.java

```diff
@@ -35,6 +35,9 @@
     @Description("maximum number of nodes to be deleted during one iteration")
     public static final Setting<Long> apoc_ttl_limit = newBuilder(APOC_TTL_LIMIT, LONG, 1000l ).build();
 
+    @Description("The batchSize applied to apoc.periodic.iterate during removing nodes")
+    public static final Setting<Long> apoc_ttl_batch_size = newBuilder(APOC_TTL_BATCH_SIZE, LONG, 10000l ).build();
```

Comment:
```suggestion
    public static final Setting<Long> apoc_ttl_batch_size = newBuilder(APOC_TTL_BATCH_SIZE, LONG, 10000L ).build();
```

se usi la `l` si confonde con `1` alle volte usa la `L`